File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
from collections import defaultdict
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        num_map = defaultdict(int)
        res = 0 

        for num in nums:
            if not num_map[num]:
                num_map[num] = num_map[num-1] + num_map[num+1] + 1
                num_map[num - num_map[num-1]] = num_map[num]
                num_map[num + num_map[num+1]] = num_map[num]
                res = max(res, num_map[num])
        return res 

File: Data Structures & Algorithms/remove-element/submission-0.java
class Solution {
    public int removeElement(int[] nums, int val) {
        int i = 0, n = nums.length;

        while(i != nums.length - 1) {
            if (nums[i] == val) {
                nums[i] = nums[--n];
            } else {
                i++;
            }
        }

        return n;
    }
}